cmd/fluxctl/sync_cmd.go

```diff
@@ -45,6 +45,8 @@ func (opts *syncOpts) RunE(cmd *cobra.Command, args []string) error {
 		return fmt.Errorf("no git repository is configured")
 	case git.RepoReady:
 		break
+	case git.RepoUnreachable:
+		return fmt.Errorf("can't connect to git repo with url %s, please check your DNS or repository url\n\nFull error message: %v", gitConfig.Remote.URL, gitConfig.Error)
```

Comment:
```suggestion
		return fmt.Errorf("can not connect to git repository with URL %s\n\nFull error message: %v", gitConfig.Remote.URL, gitConfig.Error)
```

To make it align with other errors.

Reply:
Once taken care of, can you please also smash your commits together into one? This keeps our `master` history clean.

Reply:
Yes, I have changed the error message as you suggested.